docs/pages/docs/workflows/typescript.mdx

```diff
@@ -47,3 +49,76 @@ You can freely define the interface, but if you have your messages available loc
 3. The path of your `import` is correct.
 4. Your type declaration file is included in `tsconfig.json`.
 5. Your editor has loaded the most recent type declarations. When in doubt, you can restart.
+
+## Strict typing of global formats
+
+`next-intl` provides a way to define [global formats](/docs/usage/configuration#formats) for your application. You can provide the type of your formats similarly to how you [define the type of messages](#strict-typing-of-messages).
```

Comment:
```suggestion
`next-intl` provides a way to define [global formats](/docs/usage/configuration#formats) for your application. You can provide the type of your formats similarly to how you define the type of [messages](#strict-typing-of-messages).
```

Over time I found that reducing the amount of words that are linked increases readability a bit. There are still areas in the docs where this can be improved, this looks like a chance too.